include/cute/atom/copy_traits_xe.hpp

```diff
@@ -1512,6 +1550,24 @@ struct Copy_Traits_<XE_2D_TF32x32x16_LD_N, args_t...>
       : XE_2D_LD_Unpack<XE_2D_TF32x32x16_LD_N, args_t...>(args...) {}
 };
 
+template <class... args_t>
+struct Copy_Traits_<XE_2D_TF32x8x8_LD_T, args_t...>
+    : XE_2D_LD_Unpack<XE_2D_TF32x8x8_LD_T, args_t...> {
+  using ThrID = Layout<_16>;
+  // Map from (src-thr,src-val) to bit
+  using SrcLayout = Layout<Shape <_16, Shape <_4,  _32>>,
+                           Stride< _0, Stride<_32, _1>>>;
+  // Map from (dst-thr,dst-val) to bit
+  using DstLayout = Layout<Shape <_16, Shape <_4,  _32>>,
+                           Stride< _32, Stride<_32, _1>>>;
```

Comment:
I don't think this can be right. There's a duplicate stride, and it's also right-major.

Reply:
Yeah, this looks wrong. Although, given that it is 8x8 shape I do not know what would be the correct layouts for it. So I am open to suggestions (also, this is not my code - it is coming from PR I based this on).